lib/Doctrine/ORM/Mapping/ReflectionEnumProperty.php

```diff
@@ -6,44 +6,28 @@
 
 use BackedEnum;
 use ReflectionProperty;
-use ReturnTypeWillChange;
 use ValueError;
 
 use function array_map;
 use function get_class;
 use function is_array;
 
-class ReflectionEnumProperty extends ReflectionProperty
+final class ReflectionEnumProperty extends ReflectionProperty
```

Comment:
I've finalized the classes because they extend built-in classes. If the signatures change upstream, maintaining BC for child classes is going to be a nightmare.

Reply:
Should at some point generate a list of finalized classes, then add an `@final` annotation on every such class on 2.x?